### PR TITLE
Fix: use system setting for dns server addresses

### PR DESCRIPTION
**Pull request by rjemanuele**
This was missing from Windows

Fixes https://github.com/luvit/luvit/issues/872

### CODE REVIEW

File: deps/dns.lua [hunk 1]
```diff
@@ -20,7 +20,7 @@ limitations under the License.
 -- https://github.com/openresty/lua-resty-dns/blob/master/lib/resty/dns/resolver.lua
 
 exports.name = "luvit/dns"
-exports.version = "1.0.2"
+exports.version = "1.0.3"
```
Comment (creationix) [inline deps/dns.lua]: Since this has added functionality, but is still backwards compatible, we should bump the middle version number

Comment (rjemanuele) [inline deps/dns.lua]: Ok, will do.

File: deps/dns.lua [hunk 2]
```diff
@@ -741,7 +742,89 @@ exports.setDefaultServers = function()
...
+  local servers = {}
+
+  ffi.cdef[[
+    typedef uint32_t DWORD; //Integer
+    typedef uint32_t ULONG; //Alias
+    typedef uint32_t UINT; //Alias
+    typedef ULONG *PULONG; //Pointer
+
+    enum {
+      ERROR_SUCCESS         = 0L,
```
Comment (rjemanuele) [inline deps/dns.lua]: I am wondering why this is reported as a "redefine".

File: deps/dns.lua [hunk 3]
```diff
@@ -741,7 +742,89 @@ exports.setDefaultServers = function()
...
+      MAX_HOSTNAME_LEN      = 128,
+      MAX_DOMAIN_NAME_LEN   = 128,
+      MAX_SCOPE_ID_LEN      = 256
+    };
+
+    typedef struct {
+      char String[16];
+    } IP_ADDRESS_STRING, *PIP_ADDRESS_STRING, IP_MASK_STRING, *PIP_MASK_STRING;
+
+    typedef struct _IP_ADDR_STRING {
```
Comment (rjemanuele) [inline deps/dns.lua]: Same here, redefine for _IP_ADDR_STRING

**Comment by rphillips** [comment]
+1! nice!

**Pull request merged**